src/main/resources/templates/login.html

```diff
@@ -7,25 +7,33 @@
     <title>Paw Pals</title>
 </head>
 <body>
-    <div class="imgcarousel"></div>
+
 
     <div class="login">
         <div th:if="${p == null}" class="loginform">
-            <h1>Log in</h1>
+
+
+
+            <h1>Paw Pals Login</h1>
+
             <form action="/login" method="POST">
                 <label for="username">Username: </label><input name="username" id="username" />
                 <label for="password">Password: </label><input name="password" id="password" />
-                <input style="margin:30px 30px" type="submit" />
+                <input style="margin:auto" type="submit" />
             </form>
         </div>
+        <a th:href="@{/signup}">Not a member Sign up here!</a>
+
         <div th:if="${p != null}" class="greeting">
             <p>Welcome [[${p.name}]]</p>
             <p><a th:href="@{/myprofile}">Go to your profile</a></p>
             <p><a th:href="@{/logout}">Logout!</a></p>
         </div>
-        <div th:if="${p == null}" class="greeting">
```

Comment:
Why was the hyperlink for signing up removed??! How would anyone get to the sign up page?